packages/core/src/components/toast/toast.md

```diff
@@ -30,7 +30,14 @@ horizontally aligned along the left edge, center, or right edge of its container
 
 There are three ways to use __OverlayToaster__:
 
-1. __Recommended__: use the `OverlayToaster.create()` static method to create a new `Toaster` instance:
+1. __Recommended__: use the `OverlayToaster.createAsync()` static method to create a new `Toaster` instance:
```

Comment:
I would actually swap the recommended approach here since `OverlayToaster.create()` and `OverlayToaster.createAsync()` both leak memory and add a DOM node to the page _forever_. Thoughts?

Reply:
Ok, sure, I'm down with that, we can change the recommended approach. So we'd reverse the order of approaches: 3 becomes 1 (recommended), and 1 becomes 3. Since it seems like most people prefer having an imperative API in their application to trigger toasts rather than storing a list of Toasts in app state and rendering `<Toast>` themselves.


It's worth adding more docs under the "Static usage" section around L105 with your note about memory leaks and leaving a DOM node on the page forever. Also perhaps update the code snippets there to use `createAsync` instead of `create`?

Reply:
> Ok, sure, I'm down with that, we can change the recommended approach. So we'd reverse the order of approaches: 3 becomes 1 (recommended), and 1 becomes 3. Since it seems like most people prefer having an imperative API in their application to trigger toasts rather than storing a list of Toasts in app state and rendering `<Toast>` themselves.

I pushed up a few commits that switched 1 and 3, but decided to walk this back. The primary factor that changed my mind was that it becomes harder to pass the `OverlayToaster` ref down the React component tree. I think that will be a bigger point of confusion than I originally thought when I suggested switching the recommendations.

The latest push keeps the current recommendation since it may be the best of all options, but adds more notes around only creating one `Toaster` instance.

> It's worth adding more docs under the "Static usage" section around L105 with your note about memory leaks and leaving a DOM node on the page forever.

Can do! Added in https://github.com/palantir/blueprint/pull/6599/commits/d375696c1a25cbfa9db2cc772beb4816fce788a2.

> Also perhaps update the code snippets there to use `createAsync` instead of `create`?

Good suggestion. Pushed up a commit that switches the docs and code snippets to `createAsync` here: https://github.com/palantir/blueprint/pull/6599/commits/9b1fb9334162d6726ed0403e676de0f14afd398a